contracts/test/utils/GovFixtures.sol

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {ERC20VotesComp} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20VotesComp.sol";
+import {Core} from "../../core/Core.sol";
+import {Vm} from "./Vm.sol";
+
+/// @notice Dummy contract used to test NopeDAO and Safe proposals
+contract DummyStorage {
+    uint256 private variable = 5;
+
+    function getVariable() external view returns (uint256) {
+        return variable;
+    }
+
+    function setVariable(uint256 x) external {
+        variable = x;
+    }
+}
+
+/// @notice Setup various users with Tribe delegated to them for DAO voting
```

Comment:
Can you add a bit of description here? This utility function seems pretty useful - it'd be more use if the description was more verbose.

Reply:
Sure, done